airbyte-webapp/src/views/Connector/ServiceForm/components/Sections/ConditionSection.tsx

```diff
@@ -73,16 +74,18 @@ export const ConditionSection: React.FC<ConditionSectionProps> = ({ formField, p
       })),
     [formField.conditions]
   );
-  const label = formField.title || formField.fieldKey;
+
+  const label = (
+    <PropertyLabel property={formField} label={`${formField.title || formField.fieldKey}`} optional={false} />
+  );
 
   return (
     <GroupControls
       key={`form-field-group-${formField.fieldKey}`}
-      description={formField.description}
       fullWidthTitle
       title={
         <div className={styles.sectionTitle}>
-          {label ? <GroupLabel>{label}:</GroupLabel> : null}
+          {label ? <GroupLabel>{label}</GroupLabel> : null}
```

Comment:
Could label ever be undefined now?

Reply:
Doesn't seem like it -- I've removed the conditional here